spec/features/assets_page_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe "Assets Page", :vcr do
+  let(:arbitrary_fixed_time) { Time.zone.local(2022, 9, 5, 9, 0, 0) }
+  let(:assets_header) { "Which assets does your client have?" }
+
+  before do
+    travel_to arbitrary_fixed_time
```

Comment:
I think `arbitrary_fixed_time` is one of the best names I've encountered in a while. It captures the what and the why really nicely!